README.md

```diff
@@ -82,19 +82,32 @@ It can be either a relative or absolute path.
 ## Testing
 
 The SDK has integration tests which run against a real Chatkit server.
-In order to run them, you must provide Chatkit instance credentials in
-your Gradle `local.properties` (or as `-D` arguments to the VM executing
-the tests)
 
-*Important:* The tests may delete any and all resources associated with
-the instance you provide. Create a new one. Do not share it with any other
-use.
+The tests can be run using the Gradle `test` task with the VM options mentioned below set.
+To run the tests directly from Android Studio you will need to install the
+[Spek](https://plugins.jetbrains.com/plugin/8564-spek/) plugin, and
+[Spek Framework](https://plugins.jetbrains.com/plugin/10915-spek-framework/).
+Both plugins are needed as older tests are written using Spek1, and newer
+tests are using Spek2.
+To do this go to Android Studio -> Preferences -> Plugins ->
+search for Spek, and Spek Framework and install both plugins. If you open a Spek test file
+(e.g. RoomSpek), you should now see green play buttons to run each test
+(or a test group from a file).
+
+
+In order for the tests to pass you must provide a Chatkit instance
+credential from the [dashboard](https://dash.pusher.com/chatkit/)
+to the VM - to do this edit the run configurations -> select Spek
```

Comment:
I had a suggestion here about adding "or Gradle" and using em dash instead of minus which is correct from typographical point of view. Indeed there's a reference to running via Gradle at the top and to invoke Gradle from IDE those two system properties need to be passed to the JVM running it.

Reply:
What do you mean by adding "or Gradle" sorry? I've updated the minus to be an emdash as per the symbol from https://www.thepunctuationguide.com/em-dash.html

Reply:
I mean "select Spek or Gradle", as I suggested previously [here](https://github.com/pusher/chatkit-android/pull/146#discussion_r370222783). Depending if a user wants to run tests directly from the IDE (which is useful for running individual tests) or via Gradle, they will need to set/update their corresponding run configuration. Setting configuration just for Spek won't work for Gradle.

Reply:
We talked through this yesterday in person - I've added a section explaining to add your Chatkit test instance credentials into the global gradle.properties file.